redaxo/src/core/fragments/core/form/searchfield.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+$id = '';
+$class = 'input-group input-group-xs has-feedback form-clear-button';
+$clear = rex_i18n::msg('search_clear');
+$placeholder = ' placeholder = "' . rex_i18n::msg('search_placeholder') . '"';
+
+if ($this->id) {
+    $id = ' id = "' . $this->id .'"';
+}
+if ($this->placeholder) {
+    $placeholder = ' placeholder = "' . $this->placeholder . '"';
+}
+if ($this->setclass) {
+    $class = ' class = "' . $this->class . '"';
+}
+
+echo '<div class="'. $class . '"' . $id . '>
+      <span class="input-group-addon clear-button"><i class="fa fa-search "></i></span>
+      <input class="form-control" type="text" autofocus ' . $placeholder . '>
+      <span title="' . $clear . '" class="form-control-clear fa fa-times-circle form-control-feedback hidden"></span>
```

Comment:
Bitte keine `fa-` Icons direkt notieren

Reply:
Wie dann?

Reply:
@tbaddade leider verstehe ich nicht, wie die icons gesetzt werden. die fa-times-circle  finde ich z.B. nicht. Search schon

Reply:
Beispiel `rex-icon-search-clear` notieren und dann in dieser [Datei](https://github.com/redaxo/redaxo/blob/master/redaxo/src/addons/be_style/scss/_icon.scss) dein fa-Icon via @extend holen

Reply:
thx 👍